code_challenges/code_wars/cakes.py

```diff
@@ -0,0 +1,40 @@
+def how_many_cakes(recipe, available):
+    if len(available) < len(recipe):
+        return 0
+
+    amount_of_cakes_per_ingredients = {}
+    for key in recipe:
+        if key not in available:
+            pass
+        else:
+            amount_of_cakes_per_ingredients[key] = available[key] // recipe[key]
```

Comment:
Instead of doing nothing on the fist condition can you change it so that it's the one that does the thing and then you don't need the else.

Reply:
when i do this i get a key error on my test with no matching ingredients.

Reply:
Any time you have a condition that just `pass`es you don't need it.
```python
if key in available:
    amount_of_cakes_per_ingredients[key] = available[key] // recipe[key]
```